src/webui/btjson.cpp

```diff
@@ -199,7 +199,7 @@ class QTorrentCompare
         if (type_ == QVariant::Invalid)
             type_ = torrent1.toMap().value(key_).type();
 
-        switch (type_) {
+        switch (static_cast<QMetaType::Type>(type_)) {
```

Comment:
@glassez but QMetaType::Map doesn't exists. I'm not going to change this in this PR but can you take a look and open a new PR for that?